Resizetizer.NT/Resizer.cs

```diff
@@ -19,8 +19,7 @@ public Resizer(SharedImageInfo info, string intermediateOutputPath, ILogger logg
 
 		public SharedImageInfo Info { get; private set; }
 
-		SkiaSharpBitmapTools bmpTools;
-		SkiaSharpSvgTools svgTools;
+		SkiaSharpTools tools;
```

Comment:
I love my code re-use! Using a base type to only have a single place for resize logic and draw logic.